.planning/phases/02-context-diagnostician-contracts/02-01-PLAN.md

```
@@ -0,0 +1,484 @@
+  <read_first>
+    packages/principles-core/src/runtime-v2/context-payload.ts,
+    packages/principles-core/src/runtime-v2/error-categories.ts,
+    docs/pd-runtime-v2/history-retrieval-and-context-assembly-spec.md,
+    docs/spec/2026-04-21-diagnostician-v2-detailed-design.md,
+    docs/spec/2026-04-21-pd-runtime-protocol-spec-v1.md
+  </read_first>
+  <action>
+Rewrite context-payload.ts, converting all 8 interfaces to TypeBox schemas with Static type derivation. Follow the exact pattern from error-categories.ts (Phase 1):
+
+1. Add imports at top (replace existing, keep JSDoc header):
+   ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)